### PR TITLE
Added Malawi corrective labels 2020

### PR DESCRIPTION
**Pull request by MsPixels**

### CODE REVIEW

File: datasets.py [hunk 1]
```diff
@@ -145,6 +145,17 @@ def load_labels(self) -> pd.DataFrame:
         return df
 
 
+class MalawiCorrectiveLabels2020(LabeledDataset):
+    def load_labels(self) -> pd.DataFrame:
+        Malawi_dir = raw_dir / "Malawi_corrective_labels_2020"
+        df = pd.read_csv(Malawi_dir / "Malawi_corrective_labels_2020.csv")
+        df.rename(columns={"latitude": LAT, "longitude": LON}, inplace=True)
+        df[CLASS_PROB] = (df["True value"] == 1).astype(int)
+        df[START], df[END] = date(2020, 9, 1), date(2021, 9, 1)
```
Comment (ivanzvonkov) [inline datasets.py:154]: @MsPixels for creating the dataset please use `df[START], df[END] = date(2020, 1, 1), date(2021, 12, 31)` this way we can use either February-February or September-September at training time

Comment (MsPixels) [reply datasets.py:154]: Alright Ivan

**Comment by ivanzvonkov** [comment]
Oh one other thing, the class needs to be added to the datasets list here: https://github.com/nasaharvest/crop-mask/blob/77b581b2ec2a4a4b366f4648be57488761bda84b/datasets.py#L885

File: datasets.py [hunk 2]
```diff
@@ -883,6 +894,7 @@ def load_labels(self) -> pd.DataFrame:
     KenyaCEO2019(),
     HawaiiCorrective2020(),
     HawaiiCorrectiveGuided2020(),
+    MalawiCorrectiveLabels2020(),
```
Comment (MsPixels) [inline datasets.py:897]: Added class to dataset list

**Comment by ivanzvonkov** [comment]
@MsPixels if you look at the [test Github action](https://github.com/nasaharvest/crop-mask/actions/runs/4010562110/jobs/6887199246), you'll see:
```
FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/crop-mask/crop-mask/data/raw/Malawi_corrective_labels_2020/Malawi_corrective_labels_2020.csv'
```

Do you know how to fix this?

**Comment by MsPixels** [comment]
dvc? I'm not entirely sure @ivanzvonkov. I think I'll need some help. Thanks.

**Comment by ivanzvonkov** [comment]
image

This failure makes me think that `dvc commit` was run but `dvc push` was not run or failed. Could you run `dvc push` again locally and then we can rerun the pipeline?

**Comment by MsPixels** [comment]
I tried dvc push and I got this error again 
image

**Comment by ivanzvonkov** [comment]
> I tried dvc push and I got this error again image

What if you run `gcloud auth application-default login` again to relogin, you should have the additional permissions now.

**Comment by MsPixels** [comment]
I got this after running gloud auth 
image

**Comment by ivanzvonkov** [comment]
Earth Engine export working now!

**Comment by ivanzvonkov** [comment]
Ok I took a second look at this PR, my explanation earlier this morning was not fully correct. There's a duplicate but it's across two datasets `Malawi_FAO_corrected` and `MalawiCorrectiveLabels2020`.
The points are not identical but they get matched up to the same point:
image

I am thinking we can merge this PR and figure out how to deal with this in the future.

**Comment by MsPixels** [comment]
Alright Ivan. Thank you!

**Pull request merged**